data/decks/ryu.json

```diff
@@ -0,0 +1,177 @@
+{
+	"id": "ryu",
+	"season": 3,
+	"display_name": "Ryu",
+	"cardback": "cardback_defaultexceed.png",
+	"exceed_cost": 2,
+	"character_action_default": [{
+		"gauge_cost": 0,
+		"force_cost": 0,
+		"effect": {
+			"effect_type": "choice",
+			"choice": [
+				{ "effect_type": "advance", "amount": 1 },
+				{ "effect_type": "retreat", "amount": 1 }
+			]
+		},
+	}],
+	"character_action_exceeded": [{
+		"gauge_cost": 0,
+		"force_cost": 0,
+		"effect": {
+			"effect_type": "choice",
+			"choice": [
+				{ "effect_type": "advance", "amount": 1 },
+				{ "effect_type": "retreat", "amount": 1 }
+			],
+			"and": {
+				"effect_type": "draw",
+				"amount": 1
+			}
+		},
+	}],
+	"ability_effects": [
+		{
+			"timing": "set_strike",
+			"effect_type": "gauge_for_effect",
+			"per_gauge_effect": null,
+			"overall_effect": {
+				"effect_type": "critical"
```

Comment:
if you set "character_effect": true on this overall_effect effect, will it show up next to the facedown card letting the opponent see more clearly they've used critical?
That's what I did for like Potemkin buff or Gio buff but I don't think those were inside an optional thing.

Reply:
Yup, seems to work nicely

Reply:
make sure to put it on both the regular and exceeded versions